src/main/java/org/neo4j/tool/StoreCopy.java

```diff
@@ -113,16 +113,16 @@ private static void copyRelationships(BatchInserter sourceDb, BatchInserter targ
         while (relId <= highestRelId) {
             BatchRelationship rel = null;
             try {
-                rel = sourceDb.getRelationshipById(relId++);
-                if (ignoreRelTypes.contains(rel.getType().name().toLowerCase())) continue;
```

Comment:
Move this and and the `rel = ...` to the end of the `if` clause so outputting is always handled.